src/Options/index.js

```diff
@@ -288,6 +286,9 @@ export interface ParseServerOptions {
   /* An array of keys and values that are prohibited in database read and write requests to prevent potential security vulnerabilities. It is possible to specify only a key (`{"key":"..."}`), only a value (`{"value":"..."}`) or a key-value pair (`{"key":"...","value":"..."}`). The specification can use the following types: `boolean`, `numeric` or `string`, where `string` will be interpreted as a regex notation. Request data is deep-scanned for matching definitions to detect also any nested occurrences. Defaults are patterns that are likely to be used in malicious requests. Setting this option will override the default patterns.
   :DEFAULT: [{"key":"_bsontype","value":"Code"},{"key":"constructor"},{"key":"__proto__"}] */
   requestKeywordDenylist: ?(RequestKeywordDenylist[]);
+  /* Value for Parse Server's `dns.setDefaultResultOrder` - [NodeJS' DNS lookup](https://nodejs.org/api/dns.html#dnssetdefaultresultorderorder)
```

Comment:
Why are we adding this option? Can't this be set by the developer in code if they manually create an server app, or via the `NODE_OPTIONS` env var when starting Parse Server via the CLI?

Reply:
Thanks for the tip, I didn't know it could be set via `NODE_OPTIONS` 😊